### PR TITLE
Updated/vfcbc data

### PR DESCRIPTION
**Pull request by mmumby**
- Updated Access Roads activity to include `number of sites`
Screen Shot 2021-05-27 at 12 46 00 PM

- Updated Mechanical Trenching Activity to include `length/width/depth`
Screen Shot 2021-05-27 at 12 43 58 PM

- Updated Exploration Surface drilling to include 'drill_program'
Screen Shot 2021-05-27 at 12 43 24 PM

- Updated Blasting to include 'descrive_explosives_to_site', conditionally rendered if explosives are not stored on-site
Screen Shot 2021-05-27 at 12 44 38 PM
Screen Shot 2021-05-27 at 12 45 28 PM

### CODE REVIEW

File: migrations/sql/V2021.05.26.16.47__update_drilling_and_blasting_activity.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ALTER TABLE exploration_surface_drilling ADD COLUMN drill_program character varying(100);
```
Comment (exzizt) [inline migrations/sql/V2021.05.26.16.47__update_drilling_and_blasting_activity.sql:1]: I think as a team we agreed to not impose length restrictions on data going forward? Should we remove these and the data type is just `varchar`?

Comment (mmumby) [reply migrations/sql/V2021.05.26.16.47__update_drilling_and_blasting_activity.sql:1]: Yes, I can do that. I was following the restrictions set by vFCBC, but happy to remove.

Also, you are quick!!

Comment (mmumby) [reply migrations/sql/V2021.05.26.16.47__update_drilling_and_blasting_activity.sql:1]: Updated all to varchar;

File: migrations/sql/V2021.05.20.13.44__update_now_submissions_mechanical_trenching.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE now_submissions.mech_trenching_activity ADD COLUMN length numeric;
```
Comment (mhuseinov) [inline migrations/sql/V2021.05.20.13.44__update_now_submissions_mechanical_trenching.sql:1]: I personally prefer a non-conflict column adding
ALTER TABLE <table name> ADD column IF NOT EXISTS <column name >....

**Review by mhuseinov — COMMENTED** [review]
Nice work!
Just a small comment, other than that I'm happy to approve

File: migrations/sql/V2021.05.20.13.44__update_now_submissions_mechanical_trenching.sql (hunk 2 above)
Comment (mmumby) [reply migrations/sql/V2021.05.20.13.44__update_now_submissions_mechanical_trenching.sql]: Updated!

**Review by mhuseinov — APPROVED** [review]
🔥

**Pull request merged**